edk2toolext/nuget_publishing.py

```diff
@@ -395,6 +439,14 @@ def GatherArguments():
         parser.add_argument('--CustomLicensePath', dest="CustomLicensePath", default=None,
                             help="<Optional> If CustomLicense set in `new` phase, provide absolute path of License \
                             File to pack. Does not override existing valid license.")
+        parser.add_argument("--RepositoryType", dest="RepositoryType", help="<Optional> Repository Type",
```

Comment:
Argument group as mentioned above.

Reply:
Done in PR commit https://github.com/tianocore/edk2-pytool-extensions/pull/429/commits/2682256cc76730f18db5526ac045b621f119f4db